packages/legacy/core/App/screens/PINCreate.tsx

```diff
@@ -180,13 +185,15 @@ const PINCreate: React.FC<PINCreateProps> = ({ setAuthenticated, route }) => {
               testID={testIdWithKey('EnterOldPIN')}
               onPINChanged={(p: string) => {
                 setPINOld(p)
+                setContinueEnabled(checkEmptyPINs(p))
               }}
             />
           )}
           <PINInput
             label={t('PINCreate.EnterPINTitle', { new: updatePin ? t('PINCreate.NewPIN') + ' ' : '' })}
             onPINChanged={(p: string) => {
               setPIN(p)
+              setContinueEnabled(checkEmptyPINs(p))
```

Comment:
Should this `p` be disregarded and just include a check in `checkEmptyPINs` for `PINOld`?